src/main/java/dk/sdu/se_f22/brandmodule/stemming/ExceptionUtilities.java

```diff
@@ -8,31 +8,41 @@
 
 public class ExceptionUtilities {
     public static void createException(String name) throws SQLException {
-        String query = String.format("INSERT INTO StemmingException (name) VALUES ('%s')", name);
-        ResultSet result = DBConnection.getConnection().createStatement().executeQuery(query);
-        result.close();
+        String query = String.format("INSERT INTO stemmingexceptions (exception) VALUES ('%s')", name);
```

Comment:
Please make prepared statements for queries and not just string interpolations.

Reply:
Pis også, you right. Coming right up

Reply:
If you want it extracted out in private helper functions, see TwoWaySynonyms :)